octree/include/pcl/octree/impl/octree2buf_base.hpp

```diff
@@ -664,6 +661,9 @@ Octree2BufBase<LeafContainerT, BranchContainerT>::deserializeTreeRecursive(
   }
 
   if (binaryTreeIT_arg != binaryTreeIT_End_arg) {
+    // node bits
+    char nodeBits, recoveredNodeBits;
+
     // read branch occupancy bit pattern from vector
     nodeBits = *binaryTreeIT_arg++;
```

Comment:
Why not declare and initialize `nodeBits` directly here?

Reply:
Yeah, should have done that.
Resolved 7701603